README.md

```diff
@@ -148,6 +148,22 @@ apt::source { 'puppetlabs':
 },
 ~~~
 
+### Add an Apt source served over HTTPS
+
+~~~puppet
+include apt::transport_https
+
+apt::source { 'nodesource':
+  location => "https://deb.nodesource.com/node_0.10",
+  release  => $::lsbdistcodename,
+  repos    => 'main',
+  key      => {
+    'id'     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    'source' => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key',
+  }
+}
```

Comment:
I'm not sure what you mean… maybe we're talking about something else?

~~~ puppet
class nodejs {
  . . .
  package { ['apt-transport-https','ca-certificates']:
    ensure => present,
  }
  . . .
}

class something {
  . . .
  package { ['apt-transport-https','ca-certificates']:
    ensure => latest,
  }
  . . .
}

include ::nodejs
include ::something
~~~

That's the problem I'm running into.